applications/mlflow/Taskfile.yml

```diff
@@ -0,0 +1,804 @@
+version: '3'
+
+# MLflow CI and Local Testing Taskfile
+# This file centralizes all testing and CI tasks for the MLflow application
+
+# Common variables
+vars:
+  # Directory structure
+  CHART_DIR: ./charts
+  KOTS_DIR: ./kots
+  TESTS_DIR: ./tests
+  
+  # Testing configuration
+  NAMESPACE: mlflow
+  PORT: 5000
+  
+  # Chart configuration
+  CHARTS: mlflow infra
+  
+  # Environment detection
+  CI:
+    sh: echo "${CI:-false}"
+  
+  # Resource-related parameters (adjustable for local/CI environments)
+  TIMEOUT: '{{if eq .CI "true"}}5m{{else}}3m{{end}}'
+  WAIT_RETRIES: '{{if eq .CI "true"}}30{{else}}15{{end}}'
+  RETRY_INTERVAL: '10'
+  
+  # Helm chart versions (dynamically determined)
+  MLFLOW_VERSION:
+    sh: helm show chart ./charts/mlflow | grep '^version:' | cut -d ' ' -f 2
+  INFRA_VERSION:
+    sh: helm show chart ./charts/infra | grep '^version:' | cut -d ' ' -f 2
+    
+  # Release configuration
+  APP_NAME: diamon-mlflow
```

Comment:
It is hardcode. Should we change to `APP_NAME: '{{.APP_NAME | default "diamon-mlflow"}}'`? 
or APP_NAME: '{{.APP_NAME | default ""}}' and add check afterward

Reply:
I'll also point out that Ash made a change that I merged into wg_easy branch to move from APP_NAME to whatever it is the replicated cli is using. REPLICATED_APP or something like that? Check the latest wg-easy branch.